app/src/main/java/com/codesoom/assignment/application/AuthenticationService.java

```diff
@@ -0,0 +1,20 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.utills.JwtUtill;
+import org.springframework.stereotype.Service;
+
+/**
+ * 사용자 권한을 관리
+ */
+@Service
+public class AuthenticationService {
+    private JwtUtill jwtUtill;
```

Comment:
일단 주입받은 bean은 특별한 사정이 없다면 꼭 `final`을 붙여서 재할당을 방지해 주세요.
```suggestion
    private final JwtUtill jwtUtill;
```